### PR TITLE
#162: Storing buttons palette in a configurable json file

### PR DESCRIPTION
**Pull request by oskarhinc**

### CODE REVIEW

File: data/palettes.json [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+                    "title": "Is equal to (⌨: ?=)",
+                    "value": "$$\\overset{?}{=}$$"
+                }
+            ]
+        ]
+    },
+    {
+        "screenReadersTitle": "Fraction and other notation buttons",
+        "label": "Notations",
+        "commonClass": "taticMath btn btn-notation paletteButton",
```
Comment (pawelcieszko) [inline data/palettes.json]: do you mean staticMath?

File: scripts/paletteBuilder.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
+function initializeMathPalette() {
+    $.getJSON( "../data/palettes.json", function(palettes) {
+        $('#mathPalette').html(buildMathPalette(palettes));
+    });
+}
+
+function buildMathPalette(palettes) {
+    var result = "<h3 class=\"sr-only\">math input buttons</h3>";
```
Comment (pawelcieszko) [inline scripts/paletteBuilder.js]: lowercase?

File: Debug/index.html [hunk 3]
```diff
@@ -40,6 +40,7 @@
     </script>
     <script data-main="main" src="../../mathlive/node_modules/requirejs/require.js"></script>
     <script src="../scripts/scripts.js"></script>
+    <script src="scripts/paletteBuilder.js"></script>
```
Comment (pawelcieszko) [inline Debug/index.html]: please use ../scripts/paletteBuilder.js

**Comment by pawelcieszko** [comment]
resolve conflicts

File: scripts/paletteBuilder.js (hunk 2 above)
Comment (oskarhinc) [reply scripts/paletteBuilder.js]: Yes, lowercase is used to all labels and I decided not to change this.

File: data/palettes.json (hunk 1 above)
Comment (oskarhinc) [reply data/palettes.json]: Yes, thanks

File: Debug/index.html (hunk 3 above)
Comment (oskarhinc) [reply Debug/index.html]: Ok

**Pull request merged**